index.html

```diff
@@ -7,7 +7,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Mobile First</title>
     <link rel="stylesheet" href="style.css">
-</head>
+</head> 
```

Comment:
- [ ] Great job working on the design but I think in the bottom of the page you need to fix it by adding some margin/padding to it so kindly stick with the figma as much as possible as required 🙏🏻😊🔥 and remeber to make the border as curved in the side😊🔥🔥

- Your design
![image](https://user-images.githubusercontent.com/58553711/198170132-6a28c70e-0b6d-432e-b09f-9c8278c3a29b.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/198170187-18e17a29-6729-470c-99e9-34c6545ed6a2.png)